src/components/Calculator.js

```diff
@@ -0,0 +1,25 @@
+/* eslint-disable react/prefer-stateless-function */
+/* eslint-disable react/destructuring-assignment */
+/* eslint-disable react/prop-types */
```

Comment:
- [ ] [ OPTIONAL]  KIndly fix linter issue manually after removing these lines. This is going mandatory requirement in further process.